Documentation/HelpInside/KeyboardCommands/Index.rst

```diff
@@ -77,3 +83,50 @@ Opening the backend search modal
     ..  group-tab:: macOS
 
         * :kbd:`cmd (⌘)` + :kbd:`k`
+
+
+Multiselect in content elements
+===============================
+
+..  versionadded:: 12.3
+
+The keyboard commands can be used on a select element:
+
+..  figure:: /Images/ManualScreenshots/ContentElements/SelectMultipleSideBySide.png
+    :alt: A multiselect element
+    :class: with-shadow
+
+    A multiselect element
+
+Or a folder element:
+
+..  figure:: /Images/ManualScreenshots/ContentElements/Folder.png
+    :alt: A folder element
+    :class: with-shadow
+
+    A folder element
+
+Or a group element:
+
+..  figure:: /Images/ManualScreenshots/ContentElements/Group.png
+    :alt: A group element
+    :class: with-shadow
+
+    A group element
+
+Selecting and deselecting options with the keyboard:
+
+*   :kbd:`enter` = Add options, either from right to left or left to right
+*   :kbd:`delete` or :kbd:`backspace` = Remove an option for Windows and Mac users
+*   :kbd:`alt` + :kbd:`arrow up` = Move the option one up
+*   :kbd:`alt` + :kbd:`arrow up` = Move the option one up
+*   :kbd:`alt` + :kbd:`arrow down` = Move the option one down
```

Comment:
Would have to look up if this is done via cmd on mac?

Reply:
cmd is equivalent to ctrl -> so it is safe